charts/templates/statefulset.yaml

```diff
@@ -31,83 +31,29 @@ spec:
               {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
             {{- end }}
           env:
-            - name: ML_ADMIN_USER
+            - name: MARKLOGIC_ADMIN_USERNAME
               valueFrom:
                 secretKeyRef:
                   name: {{ include "marklogic.fullname" . }}
                   key: username
-            - name: ML_ADMIN_PASS
+            - name: MARKLOGIC_ADMIN_PASSWORD
               valueFrom:
                   secretKeyRef:
                     name: {{ include "marklogic.fullname" . }}
                     key: password
+            - name: MARKLOGIC_INIT
```

Comment:
You will also need the `MARKLOGIC_JOIN` variable set to true and the bootstrap hostname set to the first host in the replicaset. 

This will take a bit of thinking as the first host should not join to itself. That's why the 

```
if [ "$HOSTNAME" != "$ML_BOOTSTRAP_HOST" ]; then
```

condition is there. You might have to add a check to the docker script that checks if the bootstrap hostname provided is the same as the local host name and not join in that case.

Reply:
Agreed, I just kept that condition in the new docker script.